packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -402,6 +408,28 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     }, leaveTouchDelay);
   };
 
+  React.useEffect(() => {
+    if (!open) {
+      return undefined;
+    }
+
+    /**
+     * @param {KeyboardEvent} nativeEvent
+     */
+    function handleKeyDown(nativeEvent) {
+      // IE 11, Edge (prior to using Bink?) use 'Esc'
+      if (nativeEvent.key === 'Escape' || nativeEvent.key === 'Esc') {
```

Comment:
Why not use `nativeEvent.keyCode` as in the other places?

Reply:
[`keyCode` is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode)

Reply:
Could we update the other cases in this case?

Reply:
Go for it if you want to. It's just deprecated and unlikely to ever be removed.

Reply:
Ok, I will try to do that in a batch, it should be quick :)